tests/fsm/test_parsing.py

```diff
@@ -9,7 +9,14 @@
 from outlines.fsm.parsing import PartialLark, PartialPythonIndenter
 
 
-def test_partial_parsing():
+@pytest.fixture
+def cleanup_lark_import():
```

Comment:
This change isn't directly related to the PR, but fixes an issue - if the tests in `test_parsing.py` fail, it cascades and causes a large number of unrelated tests to fail.